cmd/tracee-ebpf/internal/printer/printer.go

```diff
@@ -247,6 +247,7 @@ func (p *gobEventPrinter) Init() error {
 	gob.Register(make(map[string]string))
 	gob.Register(trace.PktMeta{})
 	gob.Register([]trace.HookedSymbolData{})
```

Comment:
Is this one still relevant from another event?

Reply:
yes - `hooked_syscalls`
If you could merge quickly that would be great

Reply:
> yes - `hooked_syscalls` If you could merge quickly that would be great

Exactly why I wanted to make sure before :)